lib/trento_web/openapi/schema/host.ex

```diff
@@ -0,0 +1,97 @@
+defmodule TrentoWeb.OpenApi.Schema.Host do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule IPv4 do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "IPv4",
+      type: :string,
+      format: :ipv4
+    })
+  end
+
+  defmodule IPv6 do
+    @moduledoc false
+
+    OpenApiSpex.schema(%{
+      title: "IPv6",
+      type: :string,
+      format: :ipv6
+    })
+  end
+
+  defmodule Item do
```

Comment:
Maybe call it `Host` or `HostItem` better?

Reply:
going with `HostItem` :+1: